keps/sig-scheduling/1258-default-pod-topology-spread/README.md

```diff
@@ -276,11 +277,16 @@ To ensure this feature to be rolled out in high quality. Following tests are man
 
 #### Beta (v1.20):
 
-- [ ] Finalize implementation:
-  - [ ] Map `SelectorSpreadingPriority` to `PodTopologySpread` when using Policy API.
-  - [ ] Provide knob for disabling the k8s default constraints.
-- [ ] Integration tests.
-- [ ] Verify conformance tests passing.
+- [X] Finalize implementation:
+  - [X] Map `SelectorSpreadingPriority` to `PodTopologySpread` when using Policy API.
+  - [X] Provide knob for disabling the k8s default constraints.
+- [X] Integration tests.
+- [X] Verify conformance tests passing.
+
+#### Stable (v1.24):
+
+- [X] No negative feedback.
```

Comment:
Perhaps you can list the bugs that were resolved. I think this is the only one: https://github.com/kubernetes/kubernetes/issues/102136

There is an E2E test, but I'm not sure if it runs in any CI job: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/scheduling/ubernetes_lite.go#L42

I asked in slack: https://kubernetes.slack.com/archives/C09QZ4DQB/p1641828838008400

Reply:
Let's get the test in the testgrid as part of the graduation:

we need (from discussion with @aojea):
- remove this skipper  e2eskipper.SkipUnlessProviderIs("gce", "gke", "aws")  , it will skip if there are not at least 2  zones https://github.com/kubernetes/kubernetes/blob/f86ee5649b2e127a6471bbc896614ecd32ff57c7/test/e2e/scheduling/ubernetes_lite.go#L48
- submit a PR to test infra to include the Multi-AZ Clusters to the Focus regex in one of these jobs https://github.com/robscott/test-infra/blob/master/config/jobs/kubernetes/sig-network/sig-network-kind.yaml (exact one to be confirmed)

@gavinfish can you take it?

Reply:
I actually created the PR for the first action: kubernetes/kubernetes#107472

@aojea will take care of setting up a job.

Reply:
I am not familiar with it, but will take a look first.

Reply:
It seems already worked https://testgrid.k8s.io/sig-scheduling#sig-scheduling-kind,%20multizone&include-filter-by-regex=Multi-AZ?

Reply:
yes! please include it in the KEP PR

Reply:
updated